lib/src/flutter_uploader.dart

```diff
@@ -40,56 +43,67 @@ class FlutterUploader {
   /// stream to listen on upload progress
   ///
   Stream<UploadTaskProgress> get progress {
-    return _progressChannel
-        .receiveBroadcastStream()
-        .map<Map<String, dynamic>>((event) => Map<String, dynamic>.from(event))
-        .transform(StreamTransformer<Map<String, dynamic>,
-            UploadTaskProgress>.fromHandlers(
-      handleData:
-          (Map<String, dynamic> map, EventSink<UploadTaskProgress> sink) {
-        String id = map['taskId'];
-        int status = map['status'];
-        int uploadProgress = map['progress'];
-        final t = UploadTaskProgress(
-            id, uploadProgress, UploadTaskStatus.from(status));
-
-        sink.add(t);
-      },
-    ));
+    if (_progressStream == null) {
```

Comment:
I think you can use ??=: 

```
return _progressStream ??= _progressChannel...
```

Reply:
Seems to work 👍, didn't know this null safety operator